### PR TITLE
feat: Login screen

### PR DESCRIPTION
**Pull request by tormodAase**

### CODE REVIEW

File: packages/navigation/src/_internal/getCustomRenderFunction.tsx [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
 /**
  * Custom render function that displays a custom sub-header
  * @param originalRender the original render function, found in the descriptor
  * @returns new render function
  */
-export const getCustomRenderFunction = (
+export const getCustomRenderFunction = <T extends MadBaseOptions>(
     originalRender: () => ReactNode,
+    options: T,
     CustomSubHeader?: () => ReactNode,
```
Comment (dansvend) [inline packages/navigation/src/_internal/getCustomRenderFunction.tsx:12]: Should `options` and `CustomSubHeader` be added to JSDoc?

File: packages/navigation/src/_internal/shouldDisplayCustomSubHeader.ts [hunk 2]
```diff
@@ -1,18 +1,12 @@
-import { resolveOptions } from "./resolveOptions";
-import { MadBaseOptions, MadDescriptorBase, UnresolvedScreenOptions } from "./types";
+import { MadBaseOptions } from "./types";
 
 /**
  * Calculate whether the custom sub-header should be displayed or not
  * @param descriptor descriptor from the descriptors object provided by React Navigation
  * @param unresolvedScreenOptions unresolved screen options provided by React Navigation
```
Comment (dansvend) [inline packages/navigation/src/_internal/shouldDisplayCustomSubHeader.ts:6]: Should `descriptor` and `unresolvedScreenOptions` be replaced by `options` in the JSDoc?

File: packages/core/src/components/screens/LoginScreen.tsx [hunk 3]
```diff
@@ -13,57 +13,82 @@ export const LoginScreen = () => {
     const dictionary = useDictionary();
     const authConfig = useAuthConfig();
     const navigate = useNavigateFromLoginScreen();
-    const { title, logo } = useLoginScreenConfig();
+    const { splash } = useLoginScreenConfig();
     const [demoPressCount, setDemoPressCount] = useState(0);
     const shouldDisplayDemoButton = demoPressCount >= 5;
+    const resizeMode = Platform.OS === "web" ? "contain" : "cover";
```
[the remark this answers is not in the document]
Comment (tormodAase) [reply packages/core/src/components/screens/LoginScreen.tsx:19]: It works the same for all platforms, which is the problem. Websites are usually quite wide. 
Contain:
image.png

Cover:
image.png

File: packages/core/src/components/screens/LoginScreen.tsx [hunk 4]
```diff
@@ -13,57 +13,82 @@ export const LoginScreen = () => {
...
 };
 
-const theme = EDSStyleSheet.create(theme => ({
+const theme = EDSStyleSheet.create(() => ({
     container: {
-        paddingVertical: theme.spacing.container.paddingVertical,
-        paddingHorizontal: theme.spacing.container.paddingHorizontal,
+        flex: 1,
         height: "100%",
+        backgroundColor: "#E6FAEC",
```
[the remark this answers is not in the document]
Comment (tormodAase) [reply packages/core/src/components/screens/LoginScreen.tsx:78]: The constants are not exported from `@equinor/mad-components`, and should rarely be used directly anyway. Do we really want to maintain a separate constant object for chronicles?

File: packages/core/src/components/screens/LoginScreen.tsx [hunk 5]
```diff
@@ -13,57 +13,82 @@ export const LoginScreen = () => {
...
     );
 };
 
-const theme = EDSStyleSheet.create(theme => ({
+const theme = EDSStyleSheet.create(() => ({
     container: {
-        paddingVertical: theme.spacing.container.paddingVertical,
-        paddingHorizontal: theme.spacing.container.paddingHorizontal,
+        flex: 1,
         height: "100%",
```
[the remark this answers is not in the document]
Comment (tormodAase) [reply packages/core/src/components/screens/LoginScreen.tsx:77]: Fixed

File: packages/navigation/src/_internal/getCustomRenderFunction.tsx (hunk 1 above)
Comment (tormodAase) [reply packages/navigation/src/_internal/getCustomRenderFunction.tsx:12]: Fixed

File: packages/navigation/src/_internal/shouldDisplayCustomSubHeader.ts (hunk 2 above)
Comment (tormodAase) [reply packages/navigation/src/_internal/shouldDisplayCustomSubHeader.ts:6]: Fixed

File: packages/core/src/components/screens/LoginScreen.tsx [hunk 6]
```diff
@@ -13,57 +13,82 @@ export const LoginScreen = () => {
...
-                        variant="outlined"
-                        onPress={() => {
-                            void track(metricKeys.AUTHENTICATION_DEMO);
-                            enableDemoMode();
+                <View style={styles.secretHitboxContainer}>
+                    <Pressable
+                        style={{
+                            width: "100%",
+                            height: "100%",
+                        }}
```
[the remark this answers is not in the document]
Comment (tormodAase) [reply packages/core/src/components/screens/LoginScreen.tsx:33]: Fixed

File: packages/navigation/src/_internal/getCustomRenderFunction.tsx [hunk 7]
```diff
@@ -1,21 +1,43 @@
...
+const WithCustomSubHeader = ({
+    CustomSubHeader,
+    shouldFloat,
+    children,
+}: WithCustomSubHeaderProps) => (
+    <>
+        {CustomSubHeader && !shouldFloat && <CustomSubHeader />}
+        {children}
+        {CustomSubHeader && shouldFloat && (
+            <View style={{ position: "absolute", top: 0, width: "100%" }}>
```
[the remark this answers is not in the document]
Comment (tormodAase) [reply packages/navigation/src/_internal/getCustomRenderFunction.tsx]: Fixed

File: packages/core/src/components/screens/LoginScreen.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (dansvend) [reply packages/core/src/components/screens/LoginScreen.tsx:70]: Okay - we probably don't 👍

**Review by dansvend — APPROVED** [review]
Looks good to me 👌 Good job 💪

**Pull request merged**